tests/functions.py

```diff
@@ -43,3 +46,58 @@ def function_not_enabled(a: int, b: str) -> None:
     :param b: This is another parameter
     """
     pass
+
+
+@GPTEnabled
+def function_literal(a: typing.Literal[0, 1, 2], b: typing.Literal["a", "b", "c"] = "a"):
+    """
+    This is a test function.
+
+    :param a: This is a parameter
+    :param b: This is another parameter
+    """
+    return a, b
+
+
+@GPTEnabled
+def function_add_enum(a: str):
+    """
+    This is a test function.
+
+    :param a: This is a parameter
+    """
+    return a
+
+
+function_add_enum.schema.add_enum("a", ["YES", "NO", "MAYBE"])
```

Comment:
I tend to prefer removing the spaces on lines 70 and 71 and then ignoring the error.

I think the following will do the trick
```
# type: ignore # noqa: E305;
```

Reply:
You mean one space before and one after
```python
@GPTEnabled
def function_add_enum(a: str):
    """
    This is a test function.

    :param a: This is a parameter
    """
    return a

function_add_enum.schema.add_enum("a", ["YES", "NO", "MAYBE"])

class CustomEnum(Enum):
```
or no spaces before?
```python
@GPTEnabled
def function_add_enum(a: str):
    """
    This is a test function.

    :param a: This is a parameter
    """
    return a
function_add_enum.schema.add_enum("a", ["YES", "NO", "MAYBE"])


class CustomEnum(Enum):
```

Reply:
The second one.

Reply:
I'm trying to do it, but even though you can ignore flake8, you cannot [disable black](https://stackoverflow.com/questions/58584413/black-formatter-ignore-specific-multi-line-code) unless you add two comments before and after, which is quite intrusive:
```python
# fmt: off
@GPTEnabled
def function_add_enum(a: str):
    """
    This is a test function.

    :param a: This is a parameter
    """
    return a
function_add_enum.schema.add_enum("a", ["YES", "NO", "MAYBE"])
# fmt: on
```

Reply:
Ah yeah. I think I just removed `black` from the CI haha and just kept `flake8`.